src/s1reader/version.py

```diff
@@ -4,6 +4,7 @@
 # release history
 Tag = collections.namedtuple('Tag', 'version date')
 release_history = (
+    Tag('0.2.4', '2024-03-11'),
```

Comment:
The current RTC-S1 SAS uses s1reader `0.2.2`. Maybe we should think about a version number like `0.2.2.1` to minimize the changes in the patch release.

Reply:
Let's keep the current versioning scheme please.